src/eCloud/Entities/Image.php

```diff
@@ -10,6 +10,7 @@
  * @property string $description
  * @property string $logoUri
  * @property string $documentationUri
+ * @property string $imageSize
```

Comment:
Shouldn't this be defined as an integer?

```suggestion
 * @property integer $imageSize
```